resources/references/adr/checkout/2021-03-24-nested-line-items.md

```diff
@@ -67,6 +67,7 @@ class PluginCartProcessor implements CartProcessorInterface
          */
         foreach ($lineItems as $lineItem) {
             $this->calculate($lineItem, $original, $context, $behavior, $data);
+            $toCalculate->add($lineItem);
```

Comment:
The upstream of these files in the docs is in shopware/platform. Can you change it there? 

https://github.com/shopware/platform/blob/trunk/adr/checkout/2021-03-24-nested-line-items.md

Reply:
Done https://github.com/shopware/platform/pull/2912